scripts/gradio_app.py

```diff
@@ -1225,6 +1232,15 @@ def load_example_scene(scene_name, examples_dir="examples"):
                 )
                 with gr.Column():
                     gr.Markdown("### Pointcloud Options: (live updates)")
+                    conf_thres = gr.Slider(
+                        minimum=0,
+                        maximum=100,
+                        value=50.0,
+                        step=0.1,
+                        label="Confidence Threshold Percentile (mask points below this)",
```

Comment:
maybe write: _"Confidence Threshold Percentile (mask **3D points** below this)"_
so users know that this only works in the 3D view, and not for depth&normals.